dataprofiler/profilers/profiler_options.py

```diff
@@ -543,6 +572,17 @@ def _validate_helper(self, variable_path='TextOptions'):
             errors.append("{}.vocab must be a BooleanOption."
                           .format(variable_path))
         errors += self.vocab._validate_helper(variable_path + '.vocab')
+        """
+        if self.properties["num_zeros"].is_enabled:
+            errors.append("{}: num_zeros should always be disabled,"
+                          " but was found enabled"
+                          .format(variable_path))
+
+        if self.properties["num_negatives"].is_enabled:
+            errors.append("{}: num_negatives should always be disabled,"
+                          " but was found enabled"
+                          .format(variable_path))
+        """
```

Comment:
since we are going to immediately do this next, just remove it and keep a copy. However, next time it should be a comment as in `#`